chapter_linear-regression/synthetic-regression-data.md

```diff
@@ -186,6 +209,35 @@ def get_tensorloader(self, tensors, train, indices=slice(0, None)):
         dataset = torch.utils.data.TensorDataset(*tensors)
         return torch.utils.data.DataLoader(dataset, self.batch_size,
                                            shuffle=train)
+    if tab.selected('jax'):
+        # Use PyTorch Dataset and Dataloader
+        # JAX or Flax do not provide any dataloading functionality
+        from torch.utils import data
```

Comment:
`data` is too common and it may conflict with others. So let's do `import torch` and `torch.utils.data.Dataset` (see https://github.com/d2l-ai/d2l-en/blob/master/d2l/torch.py#L244). Besides, we add import statement either at the Preface section (to be saved in d2l libraries) or at the beginning of each section.

Reply:
Yes I plan to get rid of this, earlier the idea was to isolate all the pytorch imports only in the dataset functions so that readers aren't confused. But that will not be possible, we will have to import torch and torchvision. I'll send a PR to fix this.